### DISCUSSION TITLE
Data Tiering / General - Feature Question

### DISCUSSION THREAD

**Discussion by Git-Me-Some-Hub** [Security-Onion-Solutions/securityonion#4243] [Unsupported Versions]
I am trying to evaluate the differences between SO and ElasticSIEM and I am first wondering generally wondering if SO can do basically all things that ElasticSIEM can do? 

My current specific question is around Data Tiering and how that looks and is setup exactly. For Elastic it seems it would mean a node for each data tier (hot, warm, cold), with probably a backup. Would I need a distributed node for each of those, and how do you handle redundancy? I am just seeing my kind of low key project balloon into like a 3-6 node system, and I am making sure I am understanding this correctly. 

If I know that SO is pretty much a feature equivalent, then it is just finding the configurations to carry out the end goal. Which I can do. 

Thanks as always.

**Comment by rwaight** [comment]
I have been looking for a similar mechanism to deploy 'warm' and 'cold' ES nodes within Security Onion using 'node roles'.  I'm also trying to avoid unsustainable customization of SO.  (is this related to your other [question about data tiers](https://github.com/Security-Onion-Solutions/securityonion/discussions/4537#discussion-3413485) ?)

I wonder if a sustainable way to deploy this might be to deploy an `Other` node configuration (from the install menu) with the option to deploy a 'warm' or 'cold' Elasticsearch data node within SO.  For reference, the ['Data lifecycle management with data tiers' blog](https://www.elastic.co/blog/elasticsearch-data-lifecycle-management-with-data-tiers) covers this use-case.

**Reply by TOoSmOotH** [reply]
The upcoming changes to curator in .80 will address a lot of this. You will be able to set per index how you want to handle indices in "true cluster" mode. Check out #5362 and from that you should be able to deduce the requirements to enable a warm node in elasticsearch:node_route_type on the minion pillar.

**Reply by Git-Me-Some-Hub** [reply]
Amazing news. Thanks Guys!
